File: app/src/main/java/com/example/push/MessagingService.java
package com.example.push;

import com.example.android.utils.LogUtility;
import com.google.firebase.messaging.FirebaseMessagingService;
import com.google.firebase.messaging.RemoteMessage;

public class MessagingService extends FirebaseMessagingService {
	@Override
	public void onMessageReceived(RemoteMessage remoteMessage) {
		LogUtility.d(remoteMessage.getFrom());
	}
}

File: app/src/main/kotlin/br/com/vitorsalgado/example/services/push/MessagingService.kt
package br.com.vitorsalgado.example.services.push

import br.com.vitorsalgado.example.utils.LogUtility
import com.google.firebase.messaging.FirebaseMessagingService
import com.google.firebase.messaging.RemoteMessage

class MessagingService : FirebaseMessagingService() {
  override fun onMessageReceived(message: RemoteMessage?) {
    with(LogUtility) {
      d(message?.messageId!!)
      d(message.from!!)
      d(message.to!!)
      d(message.collapseKey!!)
      d(message.messageType!!)
      d(message.sentTime.toString())
    }
  }
}
